templates/BUILD.tpl

```diff
@@ -298,49 +298,6 @@ filegroup(
     srcs = ["bin/llvm-ar"],
 )
 
-filegroup(
-    name = "as",
```

Comment:
These targets are publicly visible, so this would may be a breaking change for users that e.g. use these targets in their genrules. I think it's fine to keep these targets around for, say, the occasional `nm`-based `sh_test` of build artifacts.

Reply:
You are right. Reverted that commit